tokens.html

```diff
@@ -0,0 +1,313 @@
+		<pre><code class="language-css">body {
+	color: red;
+	line-height: normal;
+}</code></pre>
+		<pre><code class="language-json">{
+	"data": {
+		"labels": [
+			"foo",
+			"bar"
+		],
+		"series": [
+			[ 0, 1, 2, 3 ],
+			[ 0, -4, -8, -12 ]
+		]
+	},
+	// we even support comments
+	"error": null,
+	"status": "Ok"
+}</code></pre></dd>
+
+		<dt><code>punctuation</code></dt>
+		<dd>Punctuation that is not contained within a string, such as brackets.
+		<pre><code class="language-python">def median(pool):
+	'''Statistical median to demonstrate doctest.
+	>>> median([2, 9, 9, 7, 9, 2, 4, 5, 8])
+	7
+	'''
+	copy = sorted(pool)
+	size = len(copy)
+	if size % 2 == 1:
+		return copy[(size - 1) / 2]
+	else:
+		return (copy[size/2 - 1] + copy[size/2]) / 2</code></pre></dd>
+
+		<dt><code>important</code></dt>
+		<dd>Anything that is important.
+		<pre><code class="language-css">body {
+	color: red !important;
+}</code></pre>
+		<pre><code class="language-markdown"># This is a heading. Headings are important.</code></pre></dd>
+
+		<dt><code>comment</code></dt>
+		<dd>Code comments.
+		<pre><code class="language-markup">&lt;!-- Here's a comment --&gt;
+&lt;style&gt;
+	/* Here's another comment */
+&lt;/style&gt;
+&lt;script&gt;
+// Here's yet another comment
+&lt;/script&gt;</code></pre></dd>
+
+		<dt><code>inserted</code></dt>
+		<dd>Added or modified line, mainly for diffs. In general, also the idea of something being increased.
+		<pre><code class="language-diff">+		// Calculate the area of the circle
+		let area = PI * (radius ** 2);</code></pre></dd>
+
+		<dt><code>deleted</code></dt>
+		<dd>Deleted line, mainly for diffs. In general, also the idea of something being decreased/removed.
+		<pre><code class="language-diff">-		// TODO: Remove this comment
+		let foo = 'bar';</code></pre></dd>
+
+		<dt><code>bold</code></dt>
+		<dd>Bolded text. Mostly found in document-markup languages.
+		<pre><code class="language-markdown">**I am bolded text!**</code></pre></dd>
+
+		<dt><code>italic</code></dt>
+		<dd>Italicised text. Mostly found in document-markup languages.
+		<pre><code class="language-markdown">*I am italicised text!*</code></pre></dd>
+
+		<dt><code>tag</code></dt>
+		<dd>The markup version of a keyword, to indicate when an element begins and ends.
+		<pre><code class="language-markup">&lt;p&gt;Hello World!&lt;/p&gt;</code></pre></dd>
+
+		<dt><code>attr-name</code></dt>
+		<dd>Kind of like a property of a markup tag.
+		<pre><code class="language-markup">&lt;video allowfullscreen controls&gt;
+	&lt;source src="hello_world.mp4" type="video/mp4" /&gt;
+&lt;/video&gt;</code></pre></dd>
+
+		<dt><code>attr-value</code></dt>
+		<dd>The value or argument that is passed to a property in a markup tag.
+		<pre><code class="language-markup">&lt;p id="greeting"&gt;Hello World!&lt;/p&gt;</code></pre></dd>
+
+		<dt><code>namespace</code></dt>
+		<dd>Used to provide uniquely named elements and attributes in XML documents.
```

Comment:
This is also used by e.g. Java for package names.

Examples: [C#](https://prismjs.com/test.html#language=csharp&text=namespace%20Foo.Bar%20%7B%7D%0Ausing%20Foo.Bar%3B), [Java](https://prismjs.com/test.html#language=java&text=%2F%2F%20Java%0Aclass%20Foo%20extends%20foo.bar.Foo%20%7B%0A%09java.util.List%3Cfoo.bar.Foo.Bar%3E%20bar(foo.bar.Baz%20bat)%20throws%20java.lang.IOException%20%7B%0A%09%09throw%20new%20java.lang.UnsupportedOperationException(%22Not%20implemented%22)%3B%0A%09%7D%0A%7D), [Rust](https://prismjs.com/test.html#language=rust&text=use%20std%3A%3Acollections%3A%3AHashMap%3B%0Ause%20std%3A%3Afmt%3B%0Ause%20std%3A%3Aiter%3B%0Ause%20std%3A%3Aresult%3B%0Ause%20std%3A%3Async%3A%3AArc%3B%0A%0Ause%20regex_syntax%3A%3Ahir%3A%3A%7Bself%2C%20Hir%7D%3B%0Ause%20regex_syntax%3A%3Ais_word_byte%3B%0Ause%20regex_syntax%3A%3Autf8%3A%3A%7BUtf8Range%2C%20Utf8Sequence%2C%20Utf8Sequences%7D%3B%0A%0Ause%20crate%3A%3Aprog%3A%3A%7B%0A%20%20%20%20EmptyLook%2C%20Inst%2C%20InstBytes%2C%20InstChar%2C%20InstEmptyLook%2C%20InstPtr%2C%20InstRanges%2C%0A%20%20%20%20InstSave%2C%20InstSplit%2C%20Program%2C%0A%7D%3B%0A%0Ause%20crate%3A%3AError%3B%0A%0Atype%20Result%20%3D%20result%3A%3AResult%3CPatch%2C%20Error%3E%3B%0Atype%20ResultOrEmpty%20%3D%20result%3A%3AResult%3COption%3CPatch%3E%2C%20Error%3E%3B).

```suggestion
		<dd>Used to provide uniquely named elements and attributes in XML documents. Outside of markup languages, it is used to tokenize the package/namespace part of identifiers.
```